### PR TITLE
Fix dag schedule_interval

### PR DESCRIPTION
**Pull request by jovenc**
## What this pull request accomplishes:

- To match the dags files as the ones on Airflow currently

## What, in particular, needs to reviewed:

- Fix schedule interval 
- Remove refresh view for pull_miovision
- Update dag for pull_here

### CODE REVIEW

File: dags/pull_here.py [hunk 1]
```diff
@@ -55,12 +55,29 @@ def task_fail_slack_alert(context):
...
 
-dag = DAG('pull_here',default_args=default_args, schedule_interval='0 13 * * 1')
-# Run at 8 AM local time every monday
+dag = DAG('pull_here',default_args=default_args, schedule_interval=' 30 16 * * * ')
+#Every day at 1630
+# dag = DAG('pull_here',default_args=default_args, schedule_interval='0 13 * * 1')
+# Run at 1PM local time every monday
 
-t1 = BashOperator(
+# date_to_pull = datetime.today().date() - timedelta(days=2)
```
Comment (radumas) [inline dags/pull_here.py:63]: These comments can actually get cut

File: dags/pull_here.py [hunk 2]
```diff
@@ -55,12 +55,29 @@ def task_fail_slack_alert(context):
...
+
+# Execution date seems to be the day before this was run, so yesterday_ds_nodash
+# should be equivalent to two days ago. https://stackoverflow.com/a/37739468/4047679
+
+pull_data = BashOperator(
         task_id = 'pull_here',
-        bash_command = '/etc/airflow/data_scripts/.venv/bin/python3 /etc/airflow/data_scripts/here/traffic/here_api.py -d /etc/airflow/data_scripts/here/traffic/config.cfg', 
+        bash_command = '/etc/airflow/data_scripts/.venv/bin/python3 /etc/airflow/data_scripts/here/traffic/here_api.py -d /etc/airflow/data_scripts/here/traffic/config.cfg -s {{ yesterday_ds_nodash }} -e {{ yesterday_ds_nodash }} ', 
+        # params = {'date':date_to_pull.strftime('%Y%m%d')},
```
Comment (radumas) [inline dags/pull_here.py:71]: Cut

File: dags/pull_here.py [hunk 3]
```diff
@@ -55,12 +55,29 @@ def task_fail_slack_alert(context):
...
         dag=dag,
         )
+
+agg_tti = PostgresOperator(sql="SELECT covid.generate_citywide_tti('{{ yesterday_ds }}')",
+                           task_id='aggregate_tti',
+                           postgres_conn_id='here_bot',
+                           autocommit=True,
+                           retries = 0,
+                           dag=dag)
+
```
Comment (radumas) [inline dags/pull_here.py:82]: Shouldn't this be in the analysis refresh dag?

Comment (jovenc) [reply dags/pull_here.py:82]: oh no, I missed that. Was only looking at miovision & rescu for covid related stuff.

Comment (jovenc) [reply dags/pull_here.py:82]: Where does `yesterday_ds` come from?

**Pull request merged**